src/test/kotlin/no/nav/familie/ba/sak/behandling/fødselshendelse/FødselshendelseIntegrasjonTest.kt

```diff
@@ -0,0 +1,314 @@
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.time.LocalDate
+import java.time.LocalDate.of
+
+@SpringBootTest
+@ExtendWith(SpringExtension::class)
+@ContextConfiguration(initializers = [DbContainerInitializer::class])
+@ActiveProfiles("dev", "mock-dokgen", "mock-oauth", "mock-pdl-flere-barn")
+@Tag("integration")
+class FødselshendelseIntegrasjonTest(
+        @Autowired
+        private val stegService: StegService,
+
+        @Autowired
+        private val behandlingRepository: BehandlingRepository,
+
+        @Autowired
+        private val behandlingResultatRepository: BehandlingResultatRepository,
+
+        @Autowired
+        private val taskRepository: TaskRepository,
+
+        @Autowired
+        private val evaluerFiltreringsreglerForFødselshendelse: EvaluerFiltreringsreglerForFødselshendelse,
+
+        @Autowired
+        private val vedtakService: VedtakService,
+
+        @Autowired
+        private val persongrunnlagService: PersongrunnlagService,
+
+        @Autowired
+        private val personopplysningerService: PersonopplysningerService,
+
+        @Autowired
+        private val fagsakRepository: FagsakRepository,
+
+        @Autowired
+        private val andelTilkjentYtelseRepository: AndelTilkjentYtelseRepository
+) {
+
+    val now = LocalDate.now()
+
+    val infotrygdBarnetrygdClientMock = mockk<InfotrygdBarnetrygdClient>()
+    val infotrygdFeedServiceMock = mockk<InfotrygdFeedService>()
+    val featureToggleServiceMock = mockk<FeatureToggleService>()
+
+
+    val fødselshendelseService = FødselshendelseService(infotrygdFeedServiceMock,
+                                                        infotrygdBarnetrygdClientMock,
+                                                        featureToggleServiceMock,
+                                                        stegService,
+                                                        vedtakService,
+                                                        evaluerFiltreringsreglerForFødselshendelse,
+                                                        taskRepository,
+                                                        personopplysningerService,
+                                                        behandlingResultatRepository,
+                                                        persongrunnlagService,
+                                                        behandlingRepository)
+
+    @Test
+    fun `Fødselshendelse med flere barn som oppfyl vilkår skal håndteres riktig`() {
```

Comment:
```suggestion
    fun `Fødselshendelse med flere barn med oppfylt vilkårsvurdering skal håndteres riktig`() {
```

Kanskje også et bedre ord enn riktig? Hva skal skje?